src/components/calculator.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 /* eslint-disable react/prefer-stateless-function
  ,no-useless-constructor
```

Comment:
- [ ] Please remove all of the eslint-disable commands inside your calculator folder. The linters are there for a reason, to ensure that you follow all the best practices. Kindly refrain from using eslint-disable in all of your upcoming projects aswell.

Reply:
@zhadier 
thanks for your helpful review changes applied.